layers/+tools/restclient/packages.el

```diff
@@ -35,8 +38,29 @@
       (unless restclient-use-org
         (add-to-list 'auto-mode-alist '("\\.http\\'" . restclient-mode)))
       (spacemacs/set-leader-keys-for-major-mode 'restclient-mode
+        "n" 'restclient-jump-next
```

Comment:
Sorry, I mean't `,` instead of `SPC`. So actually, `SPC m g n` and `SPC m g p`. But I am not good at setting good key bindings, so let's wait for more input 😸